xml/Sector.xml

```diff
@@ -1,4 +1,5 @@
-<codelist name="Sector" xml:lang="en" category-codelist="SectorCategory" complete="1" embedded="0" xmlns:budget-alignment="http://iatistandard.org/activity-standard/overview/country-budget-alignment/">
+<?xml version="1.0" ?>
```

Comment:
Seems reasonable to add an XML declaration, though perhaps we should separately update all codelists?